app/models/master_file.rb

```diff
@@ -293,7 +293,7 @@ def update_progress_with_encode!(encode)
     self.duration = encode.tech_metadata[:duration] if encode.tech_metadata[:duration]
     self.file_checksum = encode.tech_metadata[:checksum] if encode.tech_metadata[:checksum]
     self.workflow_id = encode.id
-    self.workflow_name = encode.options[:preset]
+    #self.workflow_name = encode.options[:preset] #MH can switch to an error workflow
```

Comment:
I don't understand why github is showing this change here since it is already present in develop.